### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
This PR is raised to submit for code review in the Microvers dashboard of Mohammad Rafi Amin.

Description of the task to be completed 👎

I will be building my "BookStore" website for more than a whole week. below are the tasks that will be done.

Project 1: Initialize the project with components (React only).
Project 2: Add reducers and actions.
Project 3: Use Redux in React components.
Project 4: Connect to API.
**Project 5: Styling.**

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @mrkamin,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️ Kindly remove the instructions aiming to inform you below on your README file.
Screenshot 2023-03-02 at 8 22 59 PM
Screenshot 2023-03-02 at 8 23 04 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

@huseyinbiyik 
thanks for your good and valuable feedback.

**Comment by mrkamin** [comment]
> Hi @mrkamin,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> Alt Text [ ](https://camo.githubusercontent.com/999c41ff216bced575a5c7873c1c4ed04ef945c867ca87788003794670ee5aa7/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f785550476377453343556b586c766f6e75672f67697068792e676966) Alt Text [ ](https://camo.githubusercontent.com/999c41ff216bced575a5c7873c1c4ed04ef945c867ca87788003794670ee5aa7/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f785550476377453343556b586c766f6e75672f67697068792e676966)
> 
> ## Invalid Code Review Request
> ⚠️ Kindly remove the instructions aiming to inform you below on your README file. Screenshot 2023-03-02 at 8 22 59 PM Screenshot 2023-03-02 at 8 23 04 PM
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Hi, @huseyinbiyik 
thanks for your valuable feedback

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.books-item-btns {
+  display: flex;
+  gap: 1rem;
+  color: #8588cb;
+  border: none;
+  background-color: transparent;
+
+  /*  padding: 0 1.3rem; */
```
Comment (huseyinbiyik) [inline src/App.css]: Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase neat and professional.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+.bookstor-sect hr {
+  background-color: #898989;
+  margin: 1.1rem 0;
+}
+
+/* BooksCategory */
+
+.booksCategor-container {
+  display: flex;
```
Comment (huseyinbiyik) [inline src/App.css]: Kindly DRY your CSS as mentioned [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). For example, Display flex repeating in different classes for many times. Kindly use it as class attribute.

File: src/Components/NavBar.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.nav {
```
Comment (huseyinbiyik) [inline src/Components/NavBar.css]: You implemented the general layout very well 👍 But for this project, the project needs to look exactly same on the Zeplin file. Currenly almost each element on your page has a different styling from the Zeplin. To fix it, you can use the attributes provided by Zeplin to match the design. 
Screen Recording 2023-03-02 at 8 53 05 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @mrkamin,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Gitflow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mrkamin** [comment]
> Hi @mrkamin,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! Almost there [ ](https://camo.githubusercontent.com/edbd1e3e6d39b97475267aaea49f37d1576a9b49fe744fad786303284a0001ac/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f5276364747614e725262744e436f6e6c596f2f67697068792e676966) Almost there [ ](https://camo.githubusercontent.com/edbd1e3e6d39b97475267aaea49f37d1576a9b49fe744fad786303284a0001ac/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f5276364747614e725262744e436f6e6c596f2f67697068792e676966)
> 
> ## Highlights
> * Linter set up well and checks pass 🟢
> * Gitflow followed 👌
> * .gitignore used to ignore requested files 📁
> * PR description is descriptive 📝
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> * Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example,
>   👌"Add logo to navigation bar"
>   👌"Fix close button functionality on mobile navbar"
>   ❌ "Added logo to navigation bar"
>   ❌ "Added HTML"
>   _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi, @huseyinbiyik 
thanks for your valuable feedback I will take them into account in my future commit s

File: src/App.css (hunk 1 above)
Comment (mrkamin) [reply src/App.css]: Hi, @huseyinbiyik 
thanks for your valuable feedback

File: src/App.css (hunk 2 above)
Comment (mrkamin) [reply src/App.css]: > Kindly DRY your CSS as mentioned [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). For example, Display flex repeating in different classes for many times. Kindly use it as class attribute.

Hi, @huseyinbiyik 
thanks for your valuable feedback, could you please tell me more details how can I make it Dry as I think it is not possible.

File: src/Components/NavBar.css (hunk 3 above)
Comment (mrkamin) [reply src/Components/NavBar.css]: Hi, @huseyinbiyik 
thanks for your valuable feedback

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+  background-color: #0290ff;
+  border: none;
+  height: 2.1rem;
+  width: 11rem;
+  border-radius: 0.1rem;
+  cursor: pointer;
+}
+
+.bookstor-sect {
+  display: flex;
```
Comment (Ismailco) [inline src/App.css]: - [ ] As the previous reviewer mentioned kindly follow DRY practices in your code, which means you shouldn't use the same property too many times, for example, you can create a class `.flex` and add `display: flex` and add the class name to your HTML class property instead. this way you can use that class name for all similar situations (for example flex)

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @mrkamin, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @mrkamin 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@tufoinnkuo10 
thanks for your good and valuable feedback.

File: src/App.css (hunk 4 above)
Comment (mrkamin) [reply src/App.css]: Hi, @Ismailco 
thanks for your valuable feedback

**Pull request merged**